src/seedu/addressbook/AddressBook.java

```diff
@@ -124,6 +124,14 @@
     private static final String COMMAND_HELP_DESC = "Shows program usage instructions.";
     private static final String COMMAND_HELP_EXAMPLE = COMMAND_HELP_WORD;
 
+	private static final String COMMAND_SORT_WORD = "sort";
+	private static final String COMMAND_SORT_DESC = "List the persons in alphabetical order.";
+	private static final String COMMAND_SORT_EXAMPLE = COMMAND_SORT_WORD;
+	
```

Comment:
Trailing whitespace on this line (and for most of the empty lines in this PR as well). You can configure eclipse to trim trailing whitespace on save: http://stackoverflow.com/questions/1043433/how-to-auto-remove-trailing-whitespace-in-eclipse